guides/source/active_record_querying.md

```diff
@@ -1935,6 +1935,61 @@ SELECT books.* FROM books WHERE books.out_of_print = true
 
 [`unscoped`]: https://api.rubyonrails.org/classes/ActiveRecord/Scoping/Default/ClassMethods.html#method-i-unscoped
 
+### New Chains Inside Scope Block
```

Comment:
Why it is `new`?

Reply:
I think you are saying this is a collision with `Class#new`? This is meant as "new" in the English dictionary sense. I wanted to avoid "unscoped" as that's not technically correct. I didn't like "clean", "fresh" or "reset" better. Maybe "new" would read better in the title if we had a better noun replacement for "Chain"?  Otherwise, I'm not sure what else to use here without inventing more nomenclature which would obviously be the wrong. 

Ultimately, I want a link to provide people when discussing this behavior. You could argue that the original executable script in #44792 is the best place to start wrapping your head around it. So if this PR is barking up the wrong tree then feel free to revert. I'll just link back to the original issue.

Reply:
No I mean, new in relation to what? Why it is new?

```
Maybe something like this as the tile: Behavior of chaining scope with relation methods
```

Then in the test saying:

```
Since Rails X.x the behavior is...
```

My problem with this "new" is that 10 years for now it will be still there, and it isn't new anymore.

Or is "new" because we are creating new chain "behavior" inside scope block?